uastring/minutes/2022-04-06-minutes.md

```diff
@@ -0,0 +1,50 @@
+# [Minutes] UA String Reduction - 2022-04-06
+
+## Attendees:
+
+* Karl Dubost, Mozilla
+* Chris Peterson, Mozilla
+* Mike Taylor, Google
+
+### Version 💯
+* Chrome 💯 is now 💯% across all channels.
+    * https://chromiumdash.appspot.com/releases?platform=Windows
+
+* Firefox 100 in beta currently. Yoohoo!
+    * Release date: May 3.
+    * https://github.com/webcompat/web-bugs/labels/version100
+        * 27 Firefox version100 webcompat issues are open, atm: https://github.com/webcompat/web-bugs/issues?q=is%3Aopen+label%3Abrowser-firefox+label%3Aversion100
+        * 15 issues with sitepatch applied https://github.com/webcompat/web-bugs/issues?q=label%3Aversion100+label%3Asitepatch-applied
+        * only 3 issues still require a sitepatch https://github.com/webcompat/web-bugs/issues?q=is%3Aopen+label%3Aaction-needssitepatch+label%3Aversion100
+* Only a few dozen reported for now, but some concern that we didn’t learn about jQuery Reject or Yell/Duda breakage through webcompat.com issues. What unreported issues are we still missing?
+* Yottaa CDN issues?
+    * https://www.hannaandersson.com/ issue https://github.com/webcompat/web-bugs/issues/100779 looks like it might be a different problem.
+    * Test other Yottaa customers?
+    * https://www.yottaa.com/customer-stories/
+    * Some of their customers are major sites (like Shopify) so we probably would have heard if they were all broken.
+    * No override yet for the CDN https://github.com/mozilla-extensions/webcompat-addon/blob/896b9f911c8786f036ed85cc5caa259e06156406/src/data/ua_overrides.js#L716-L738
+* Any new webcompat issues?
+    * Not really
+    * [jQuery Reject sites (Chrome 100 version parsing error) - Public](https://docs.google.com/spreadsheets/d/1uk2EiDrsuqBGFTjT5QGhliAuenzDgfZD9EXZifzzEz8/edit#gid=0)
+        * about 100 sites still unfixed
+        * shows an “upgrade your browser” message to Chrome and Edge 100+
+        * Mike will do the outreach for top sites only
+
+### Next steps for Compat Standard’s UA section?
+    * https://compat.spec.whatwg.org/#ua-string-section
+    * Mike is still addressing bugs (slowly). Rewrote Safari UA section.
+    * When UA reductions ship, no need to describe the change. Describing the latest UA is OK.
```

Comment:
```suggestion
* https://compat.spec.whatwg.org/#ua-string-section
* Mike is still addressing bugs (slowly). Rewrote Safari UA section.
* When UA reductions ship, no need to describe the change. Describing the latest UA is OK.
```

to note make it a pre

Reply:
> to note make it a pre

Sorry, I don't follow. Should something be wrapped in backticks, @karlcow?

Reply:
currently the `*` has too many spaces it converts it as a pre instead of a list item.

Reply:
haha!

to **NOT** make it a pre

Reply:
![Capture d’écran 2022-04-08 à 06 25 23](https://user-images.githubusercontent.com/505230/162319721-fd592761-5801-4717-b50f-ffe568d48312.png)

Reply:
Ah! Now I see. :)

And I realize I only did the rich preview for the first doc, and not the second. Oops.